src/components/Homes/Home.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import NavBar from '../Nav/Navbar';
+import Article from '../Articles/Article';
+import Tags from '../Tag/Tags';
+import Footer from '../Footers/Footer';
+
+
+class Home extends React.Component {
+  render() {
+    return (
+      <div>
+        <NavBar />
```

Comment:
Call the `Navbar` component from `App.js` so that it is uniform on all pages.

Reply:
Resolved